### PR TITLE
update instance type picked on codedeploy test

### LINKED ISSUE #1807: Codedeploy flakiness
```
start and wait for deployment
[49609](https://github.com/iasql/iasql-engine/actions/runs/3950005244/jobs/6762033252#step:9:49610)

[49610](https://github.com/iasql/iasql-engine/actions/runs/3950005244/jobs/6762033252#step:9:49611)
    QueryFailedError: {"result":{"state":"FAILURE","reason":{"$metadata":{"httpStatusCode":200,"requestId":"00ab0b29-cad6-4f7d-bbed-bd86c8c0adf4","attempts":1,"totalRetryDelay":0},"deploymentInfo":{"applicationName":"g08wq0pcodedeploytestapplicationForDeployment","completeTime":"2023-01-18T15:44:48.016Z","computePlatform":"Server","createTime":"2023-01-18T15:39:44.063Z","creator":"user","deploymentConfigName":"CodeDeployDefault.OneAtATime","deploymentGroupName":"g08wq0pcodedeploytestdeployment_group","deploymentId":"d-Q49G7NNHL","deploymentOverview":{"Failed":1,"InProgress":0,"Pending":0,"Ready":0,"Skipped":0,"Succeeded":0},"deploymentStatusMessages":[],"deploymentStyle":{"deploymentOption":"WITHOUT_TRAFFIC_CONTROL","deploymentType":"IN_PLACE"},"errorInformation":{"code":"HEALTH_CONSTRAINTS","message":"The overall deployment failed because too many individual instances failed deployment, too few healthy instances are available for deployment, or some instances in your deployment group are experiencing problems."},"fileExistsBehavior":"DISALLOW","ignoreApplicationStopFailures":false,"instanceTerminationWaitTimeStarted":false,"relatedDeployments":{},"revision":{"gitHubLocation":{"commitId":"cf6aa63cbd2502a5d1064363c2af5c56cc2107cc","repository":"iasql/iasql-codedeploy-example"},"revisionType":"GitHub"},"rollbackInfo":{},"status":"Failed","updateOutdatedInstancesOnly":false}}}}

```

### PR DESCRIPTION
**Pull request by aguillenv**
Allegedly resolves #1807

### CODE REVIEW

File: test/modules/aws-codedeploy-integration.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const getInstanceTypeOffering = async (availabilityZones: string[]) => {
       },
       {
         Name: 'instance-type',
-        Values: ['t2.micro', 't3.micro'],
+        Values: ['t*.medium', 't*.large'],
```
Comment (dfellis) [inline test/modules/aws-codedeploy-integration.ts:60]: `t*` works? Also, it should be `m*.large` for the second one?

Comment (yrobla) [reply test/modules/aws-codedeploy-integration.ts:60]: there can be also t*.large... but we could make it wider, and use t*.medium, t*.large, m*.large ?

**Comment by aguillenv** [comment]
Apparently, there is an issue with code deploy and [AWS documentation](https://docs.aws.amazon.com/codedeploy/latest/userguide/troubleshooting-deployments.html#troubleshooting-deployments-lifecycle-event-failures) says that you could use the `ignore-application-stop-failures` option 🤔 I added it and seems to be working again.

**Comment by aguillenv** [comment]
this is not the way to go to solve this issue.

**Pull request closed without merging**